### PR TITLE
[quant][graphmode] Move clamp to general value ops map

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #38436 [quant] Histogram Observer support List[Tensor]
* #38435 [quant] Remove get_qparams in Observers
* #38434 [quant][graphmode] Add RemoveReduantDequantize pass
* #38359 [quant][graphmode][refactor] Factor out getFixedQParamOpFusionInfo
* #38358 [quant][graphmode][refactor] Factor out getInputTensorQParamOpFusionInfo
* #38278 [quant][graphmode] Support ops with fixed quantization parameters
* #38277 [quant][graphmode][refactor] insertDeQuantForAllUse
* #38276 [quant][graphmode][refactor] Refactor propagateQuantizationOps
* #38166 [quant][graphmode] Move leaky_relu to general value op map
* #38165 [quant][graphmode] Move elu to general value ops map
* #38164 [quant][graphmode] Move hardtanh to general value ops map
* **#38163 [quant][graphmode] Move clamp to general value ops map**
* #38162 [quant][graphmode] Move interpolate to general value ops

Summary:

Test Plan:

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21559805](https://our.internmc.facebook.com/intern/diff/D21559805)

### CODE REVIEW

File: torch/csrc/jit/passes/quantization/quantization_patterns.h [hunk 1]
```diff
@@ -604,6 +604,18 @@ graph(%a_quant, %size, %align_corners, %scales_d, %scales_h, %scales_w):
           %r = aten::upsample_trilinear3d(%a_quant, %size, %align_corners, %scales_d, %scales_h, %scales_w)
           return (%r) )";
 
+  // aten::clamp
```
Comment (raghuramank100) [inline torch/csrc/jit/passes/quantization/quantization_patterns.h:607]: For these types of ops, how are the scale and zero-point set in IR?

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization/quantization_patterns.h:607]: we inherit the scale/zero_point from the previous quantized tensor, the code is in the first PR in the stack

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization/quantization_patterns.h:607]: oh, actually, the code is already landed in the master

Comment (jerryzh168) [reply torch/csrc/jit/passes/quantization/quantization_patterns.h:607]: https://codebrowser.bddppq.com/pytorch/pytorch/torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp.html#643

**Review by supriyar — APPROVED** [review]

**Pull request closed without merging**